tests/unit/OfficeAddresses.spec.ts

```diff
@@ -59,8 +62,8 @@ describe('Office Address delivery address <same as> is unchecked by default', ()
 
   // the commented corp types are not available to test currently
   const CORP_TYPES = [
-    'CP',
-    'BEN'
+    CorpTypeCd.COOP,
+    CorpTypeCd.BENEFIT_COMPANY
     // 'CC',
     // 'BC',
     // 'ULC'
```

Comment:
Would you mind updating these, too, please?

Reply:
Not al all. Done. Kept them commented out but changed to use Enums

Reply:
TYVM